NorthstarDLL/server/auth/serverauthentication.cpp

```diff
@@ -62,6 +62,22 @@ void ServerAuthenticationManager::StartPlayerAuthServer()
 						return;
 					}
 
+					uint64_t uid;
+					try
+					{
+						uid = std::strtoull(request.get_param_value("id").c_str(), nullptr, 10);
+					}
+					catch (std::exception const& ex)
+					{
+						response.set_content("{\"success\":false}", "application/json");
+						return;
+					}
+					if (!g_pBanSystem->IsUIDAllowed(uid))
```

Comment:
why still call this if the strtoull call from earlier fails?

Reply:
We don't; there's a return in the try/catch.